### ISSUE TITLE
[Feature Improvement] Avoid message duplication

### ISSUE THREAD

**Issue by sarojrana** [integrations/slack#971]
### Issue
As a user I can subscribe:
* organization (automatically subscribes all repositories available)
* specific repository of same organization  

**Result:** user gets same message twice

Can subscription feature be updated so that if organization is subscribed and if user tries to subscribe specific repository then alert user as `You're already subscribed to organization/repo` ?

**Comment by alexfinnarn** [comment]
yes, this is very annoying. I guess I double subscribed, but...why would you ever want to post duplicated messages in a channel? I can't think of a reason. I now have to unsubscribe...or figure out what the hell to do, but the install instructions should be updated so this isn't so easy to do. 

Also, I think by subscribing to comments you actually subscribe to everything? since every addition to anything is considered a comment? 

Whatever the case, I only manually subscribed to comments and now get duplicate notifications for everything.

**Comment by einarpersson** [comment]
Super annoying
